### PR TITLE
Add supported dotnet versions 4.7.2 and 4.8 to Win 10.16229

### PR DESCRIPTION
**Pull request by opozot**
This small commit will add dotnet 4.7.2 and 4.8 versions to v4_helper.rb. 
This is due to our unsupported version error.

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures ms_dotnet'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '4.2.1'
+version          '4.2.2'
```
Comment (Annih) [inline metadata.rb]: Please do n ot change the metadata we'll do the proper release once your request has been merged

Comment (opozot) [reply metadata.rb]: Sure, sorry about that, reverted.

**Comment by opozot** [comment]
> LGTM
> How complexe was it to add the support for 4.8?
> Did you use https://github.com/criteo-cookbooks/ms_dotnet/wiki/Add-a-new-supported-minor-version-of-.NET-Framework?

It was not that complex, I managed to update the cookbook just by looking at the code. True, it wasn't without errors, but the wiki helped a lot.

**Comment by yamamoto52** [comment]
When it could be merged ?

**Comment by cameronkroeker** [comment]
when can this be merged? Also, can support for Windows Server 2019 be added as well?

```
# Windows Server 2019
        when '10.0.17763' then %w(4.8)
```
https://github.com/criteo-cookbooks/ms_dotnet/blob/d5df56ddd5b41fffdab5ee5572303a0e77bda253/libraries/v4_helper.rb#L130

**Comment by compuguy** [comment]
It looks like with a few additions, it should install .NET 4.8 on Windows Server 2012R2 via the offline installer.

**Comment by EAS-Rhys** [comment]
Are there plans to merge this to enable 4.8 on windows 2019 server? Seems to have gone stale

**Pull request merged**